docs/sources/enterprise/kms-integration/using-aws-kms-to-encrypt-database-secrets.md

```diff
@@ -0,0 +1,78 @@
+
+4. From within Grafana, turn on [envelope encryption]({{< relref "../../../administration/envelope-encryption.md" >}}).
+5. Add your AWS KMS details to the Grafana configuration file; depending on your operating system, it is usually named `grafana.ini`:
+ <br><br>a. Add a new section to the configuration file, with a name in the format of `[security.encryption.awskms.<KEY-NAME>]`, where `<KEY-NAME>` is any name that uniquely identifies this key among other provider keys.
+ <br><br>b. Fill in the section with the following values:
+ `key_id`: a reference to a key stored in the KMS. This can be a key ID, a key Amazon Resource Name (ARN), an alias name, or an alias ARN. If you are using an alias, use the prefix “alias/.” To specify a KMS key in a different AWS account, use its ARN or alias. For more information about how to retrieve a key ID from AWS, refer to [Finding the key ID and key ARN](https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html).
+
+Here are some examples of values you can use in the `key_id` field:
+
+Key ID: `1234abcd-12ab-34cd-56ef-1234567890ab`
+Key ARN: `arn:aws:kms:us-east-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab`
+Alias name: `alias/ExampleAlias`
+Alias ARN:  `arn:aws:kms:us-east-2:111122223333:alias/ExampleAlias`
+`access_key_id`: The AWS Access Key ID that you previously generated.
+`secret_access_key`: The AWS Secret Access Key you previously generated.
+`token`: (Optional) An AWS Session Token, which you must provide if you created temporary credentials.
+`AWS Region`: The AWS region where you created the KMS key. The region is contained in the key’s ARN. For example: `arn:aws:kms:*us-east-2*:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab`
+
+Here is an example of an AWS KMS provider section in the `grafana.ini` file:
+```
+# AWS key management service provider setup
+;[security.encryption.awskms.example-encryption-key]
+# Reference to a KMS key - either key ID, key ARN, alias name, or ARN
+;key_id = 1234abcd-12ab-34cd-56ef-1234567890ab
+# AWS access key ID
+;access_key_id = AKIAIOSFODNN7EXAMPLE
+# AWS secret access key
+;secret_access_key = wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+# AWS session token, optional
+;token = AQoDYXdzEJr...<REMAINDER OF SECURITY TOKEN>
+# AWS region, for example eu-north-1
+;region = eu-north-1
+```
+
+Update the `[security]` section of the `grafana.ini` configuration file with the new Encryption Provider key that you created:
+
+```
+[security] 
+# previous encryption key, used for legacy alerts, decrypting existing secrets or used as default provider when external providers are not configured
+secret_key = AaaaAaaa
+# encryption provider key in the format <PROVIDER>.<KEY_NAME>
+encryption_provider = awskms.example-encryption-key
```

Comment:
`encryption_provider` means current provider used for encryption.
I added one more setting to list all configured providers - `available_encryption_providers`): a user needs to put references to providers sections they configured. 

In your example,`available_encryption_providers ` should be set with the same value as `encryption_provider` - awskms.example-encryption-key. But if they have multiple providers configured, it looks like [here](https://github.com/grafana/grafana-enterprise/pull/2283#issue-782637464):